src/components/Icon.ts

```
@@ -1,638 +1,2 @@
-      }}
-      fill='none'
-      xmlns='http://www.w3.org/2000/svg'
-    >
-      {icon.content}
-    </svg>
-  );
-};
-
-export type { IconName, IconProps };
-export default Icon;
+export { default, ICON_NAMES } from "./Icon/index";
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)